imagebuildah/build.go

```diff
@@ -185,6 +185,8 @@ type BuildOptions struct {
 	Jobs *int
 	// LogRusage logs resource usage for each step.
 	LogRusage bool
+	// Excludes is the contents of the .dockerignore file.
```

Comment:
... to be used _instead_ of the contents of the .dockerignore file.  This doesn't make it clear that the caller can leave it unset to let the default thing happen.

Reply:
excellent point @nalind

Reply:
Fixed